crawl-ref/source/item-name.cc

```diff
@@ -2949,7 +2951,7 @@ bool is_useless_item(const item_def &item, bool temp, bool ident)
 
         case AMU_FAITH:
             return (you.species == SP_DEMIGOD && !you.religion)
-                    || you_worship(GOD_GOZAG)
+                    || you_worship(GOD_GOZAG) || you_worship(GOD_ASHENZARI)
```

Comment:
I still think it'd be nice to get "faith working with ash in the same way it does with Ru, but it's not a priority - we can talk about it more once the first version is out.